mongo-client/Dockerfile

```diff
@@ -1,35 +1,35 @@
 FROM ubuntu:16.04
 
-RUN apt-get clean \
-  && rm -rf /var/lib/apt/lists/* \
-  && apt-get update -y || true \
-  && apt-get install -y -f \
-  make \
-  automake \
-  libmysqlclient-dev \ 
-  libtool \
-  libsasl2-dev \
-  libssl-dev \
-  libmongoc-dev \
-  libbson-dev 
+RUN apt-get update && apt-get -y --force-yes install --no-install-recommends \
```

Comment:
@bilal-mk We figued out that **--no-install-recommends** does not allows to install some other dependency related to sysbench.